### PR TITLE
Refactor field notes

### PR DESCRIPTION
**Pull request by nlubisch**
### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code

### CODE REVIEW

File: htdocs/src/Oc/GeoCache/Persistence/GeoCacheLog/GeoCacheLogRepository.php [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+     *
+     * @param array $data
+     *
+     * @return GeoCacheLogEntity
+     */
+    public function getEntityFromDatabaseArray(array $data)
+    {
+        $entity = new GeoCacheLogEntity();
+        $entity->id = (int) $data['id'];
+        $entity->uuid = (string) $data['uuid'];
```
Comment (teiling88) [inline htdocs/src/Oc/GeoCache/Persistence/GeoCacheLog/GeoCacheLogRepository.php]: Values from the Database are normally always strings :)

File: htdocs/src/Oc/FieldNotes/Enum/LogType.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+namespace Oc\FieldNotes\Enum;
+
+use Oc\GeoCache\Enum\LogType as GeoCacheLogType;
+
+/**
+ * Class LogType
+ *
+ * @package Oc\FieldNotes\Enum
+ */
+final class LogType
```
Comment (teiling88) [inline htdocs/src/Oc/FieldNotes/Enum/LogType.php]: really?

Comment (nlubisch) [reply htdocs/src/Oc/FieldNotes/Enum/LogType.php]: removed all

File: htdocs/src/Oc/GeoCache/Persistence/GeoCacheLog/GeoCacheLogRepository.php (hunk 1 above)
Comment (nlubisch) [reply htdocs/src/Oc/GeoCache/Persistence/GeoCacheLog/GeoCacheLogRepository.php]: removed them at all repositories

**Pull request merged**